flink-runtime/src/test/java/org/apache/flink/runtime/scheduler/strategy/TestingSchedulingTopology.java

```diff
@@ -78,6 +83,31 @@ public TestingSchedulingResultPartition getResultPartition(final IntermediateRes
 		return resultPartition;
 	}
 
+	@Override
+	public Iterable<SchedulingPipelinedRegion> getAllPipelinedRegions() {
+		return vertexRegions.values().stream().collect(Collectors.toSet());
```

Comment:
I think that's the same as `new HashSet<>(vertexRegions.values())`

Reply:
Ok. Let's simplify it.